pkgs/top-level/all-packages.nix

```diff
@@ -1926,6 +1926,8 @@ with pkgs;
 
   kubevirt = callPackage ../tools/virtualization/kubevirt { };
 
+  lastversion = with python3Packages; toPythonApplication lastversion;
```

Comment:
Call me insane, but it would be interesting to put the code below in `pkgs/by-name/la/lastversion/package.nix`:

```nix
{ python3Packages }:

python3Packages.toPythonApplication python3Packages.lastversion
# or the shorter
# with python3Packages; toPythonApplication lastversion
```

and remove that line from `all-packages.nix`.

Reply:
Sounds cool!

`all-packages.nix` is a huge source of merge conflicts and undiscoverable tweaks. Let's make it lighter.

Reply:
The trade-off is that people would have to override `python3Packages` to override `pkgs.lastversion`'s dependencies, but that's actually cleaner that relying on the assumption .that they all comes from `pkgs.python3Packages`.